src/api/users/controllers/createUserProject.js

```diff
@@ -0,0 +1,20 @@
+const prisma = require("../../../../prismaClient");
+
+const createUserProject = async (req, res, next) => {
+  const { userId, projectId } = req.params;
+  try {
+    await prisma.user.update({
+      where: {
+        id: userId,
+      },
+      data: {
+        projects: { connect: { id: projectId } },
+      },
+    });
+    res.sendStatus(201).json({ message: "done for me" });
```

Comment:
Same as for the other route: explicit message please.

Reply:
ok done / in previous pr